api/src/api/teams.ts

```diff
@@ -3,6 +3,7 @@ import { errorWrap } from '../middleware';
 
 import Team from '../models/team';
 import { requireAdmin, requireRegistered } from '../middleware/auth';
+import { ITeam } from '../../../common';
```

Comment:
This should not be a relative path. Instead: `import { ITeam } from 'ssw-common';`